src/fides/api/models/pre_approval_webhook.py

```diff
@@ -1,14 +1,23 @@
-from sqlalchemy import Boolean, Column, ForeignKey, String
+from sqlalchemy import Boolean, Column, ForeignKey, String, UniqueConstraint
 from sqlalchemy.orm import Session, relationship  # type: ignore
 
 from fides.api.db.base_class import Base
 
 
 class PreApprovalWebhook(Base):
+    """
+    An HTTPS callback that calls an external REST API endpoint as soon as a privacy request
+    is received (or after user identity verification, if applicable).
+    """
+
     name = Column(String, unique=True, nullable=False)
     key = Column(String, index=True, unique=True, nullable=False)
-    connection_config_id = Column(
-        String, ForeignKey("connectionconfig.id"), nullable=False
+    connection_config_id = (
+        Column(  # Deleting a Connection Config also deletes the Pre Approval Webhook
+            String,
+            ForeignKey("connectionconfig.id", ondelete="CASCADE"),
+            nullable=False,
+        )
```

Comment:
New, if a connection config is deleted, I'm automatically deleting its PreApprovalWebhook -